packages/core/src/enums/RequestType.ts

```diff
@@ -6,8 +6,10 @@ enum RequestType {
   Interaction = 'interaction',
   /** Second highest priority for loading*/
   Thumbnail = 'thumbnail',
-  /** Lowest priority for loading*/
+  /** Third highest priority for loading, usually used for image loading in the background*/
   Prefetch = 'prefetch',
+  /** Lower priority, often used for background computations in the worker */
```

Comment:
I think I would say instead that this request type does not consume bandwidth as it is local compute.  Thus, it has a different queue to use.  It is not whether the priority is lower or higher, it is different.

Reply:
No, it is, in fact, lower in priority when queued. This should be comparing a decoding task versus a computation task.